pkg/providers/v1/aws_ec2.go

```diff
@@ -87,6 +88,21 @@ func (s *awsSdkEC2) DescribeInstances(ctx context.Context, request *ec2.Describe
 	return results, nil
 }
 
+func (s *awsSdkEC2) DescribeInstanceTopology(ctx context.Context, input *ec2.DescribeInstanceTopologyInput, optFns ...func(*ec2.Options)) ([]ec2types.InstanceTopology, error) {
```

Comment:
I see we are moving the functionality of the https://github.com/kubernetes/cloud-provider-aws/blob/master/pkg/services/aws_ec2.go to here, can we deprecate that file or is it used somewhere else?